src/dev/register_git_hook/register_git_hook.js

```diff
@@ -138,11 +138,13 @@ export async function registerPrecommitGitHook(log) {
   );
 
   try {
+    const homedir = process.env.HOME ? process.env.HOME.replace(/'/g, '\'')
```

Comment:
I'm pretty sure I have added this escaping when I was building our version of the pre-commit hook in order to prevent repeat old bugs that already happened on husky. In that case I think it was because of that one https://github.com/typicode/husky/issues/117